Changed UI of suggested_coding_site.html

abhiraj-ku opened the pull request:
# Description
Best and Free Online Websites to Learn Coding for Beginners was not looking good as all the text was in long paragraph style 
and also the background gradient colors were not complementary so it not visible also ..

Please include a summary of the change and which issue is fixed.
Fixes # (issue)
The changes made are that the paragraph style text is been changed into responsive cards design made Flex-box and CSS only


## List any dependencies that are required for this change 
No , Dependencies were required just pure CSS was applied

## Type of change
Enhancement of suggested_coding_site.html

## How Has This Been Tested?
With the help of developer tools 
## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
![before ui](https://user-images.githubusercontent.com/113116498/207777723-6c121512-41bf-40f4-ac1f-1f9855207bfe.PNG)

    - After :
![after ui](https://user-images.githubusercontent.com/113116498/207777745-0d655bf1-7c47-4ce1-af0d-8a3986a4e286.PNG)

## Checklist:

- [yes ] My code follows the style guidelines of this project
- [ yes] I have performed a self-review of my own code
- [yes] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ yes] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

abhiraj-ku commented:
hey @harikesh409 I have made a PR regarding the enhancement .waiting for your reply

harikesh409 reviewed (changes requested):
Where are the images being used?

harikesh409 commented:
<img width="1755" alt="image" src="https://user-images.githubusercontent.com/3501554/207857420-9cffcbdf-44eb-4418-9f12-9f0469b48037.png">
The contact us and the copyright footer is messed up.

abhiraj-ku commented:
> The contact us and the copyright footer is messed up.

its done .. 


![footer](https://user-images.githubusercontent.com/113116498/207896349-bbf78887-a36d-4fb7-b826-4c0f0d866baf.PNG)

abhiraj-ku commented:
Added the images and also changed the name of images folder to suggested-sites

![new bg](https://user-images.githubusercontent.com/113116498/207899786-690f9fcd-0a2b-4fd0-9058-11dd678076e7.PNG)

harikesh409 commented:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/3501554/207936170-f2200f7d-940a-4a11-9e30-7d67e4ed3468.png">
The main content is not proper now.

abhiraj-ku commented:
I have applied  the changes you suggested and its working fine on local machine . i have pushed them also so that you can see the changes

harikesh409 commented:
<img width="1788" alt="image" src="https://user-images.githubusercontent.com/3501554/208062293-20726380-8063-44cc-a787-dc2585f2bfff.png">
check for this resolution, the responsive is not proper.

abhiraj-ku commented:
Hey actually i don't have monitor of this size so i made keeping in mind of the laptops and mobile screen .. so i am having hard time visualizing it ..so could you help a bit 🙂

harikesh409 commented:
@abhiraj-ku 
You don't need to have a laptop of that size just use devtools to check those resolutions.

abhiraj-ku commented:
@harikesh409 i am actually having hard times figuring media queries for this resolution even using devtools so if you could suggest something that would be good 🙂
